apps/api/src/imbi/api/auth/internal_services.py

```
@@ -0,0 +1,409 @@
+
+INTERNAL_SERVICES: tuple[InternalService, ...] = (
+    InternalService(
+        slug='imbi-scheduler',
+        display_name='Imbi Scheduler',
+        description=(
+            'Runs scheduled tasks that target the Imbi API (ADR 0002)'
+        ),
+        role_slug='imbi-scheduler',
+        credential='client_credential',
+        client_id_var='IMBI_SCHEDULER_SA_CLIENT_ID',
+        secret_var='IMBI_SCHEDULER_SA_CLIENT_SECRET',
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to argument: "secret_var"

(S106)